src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    private static final Figure[] figures = new Figure[6];
```

Comment:
[All magic numbers in your code should be constants.](https://github.com/Cliffcamer/jv-oop-advanced/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants)